article/_posts/2019-11-21-stealml.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: review
+title: "Stealing Machine Learning Models via Prediction APIs"
+tags: machine-learning deep-learning
+author: "Antoine Théberge"
+cite:
+    authors: "Florian Tramèr, Fan Zhang, Ari Juels, Michael K. Reiter, Thomas Ristenpart"
+    title:   "Stealing Machine Learning Models via Prediction APIs"
+    venue:   "USENIX"
+pdf: "https://arxiv.org/abs/1609.02943"
+---
+
+
+# Highlights
+- Simple equation-solving model extraction attacks for simple ML model
```

Comment:
(...) simple ML model : (...) simple ML models ?

Reply:
fixed in a5fac43